modules/blocks.php

```diff
@@ -26,8 +26,10 @@ function jetpack_map_block_load_assets( $attr, $content ) {
 		'lodash',
 		'wp-element',
 		'wp-i18n',
-		'wp-api-fetch',
 	);
+
+	$api_key = Jetpack_Options::get_option( 'mapbox_api_key' );
+
 	Jetpack_Gutenberg::load_assets_as_required( 'map', $dependencies );
-	return $content;
+	return preg_replace( '/<div /', '<div data-api-key="'. $api_key .'"', $content, 1 );
```

Comment:
Rather than prepending the `api-key` attribute, what about ignoring `$content` here and replicating the entire[ `div` from the `save` method](https://github.com/Automattic/wp-calypso/blob/ad03acb6c0653fe14a3a4eb7fa8d321c7d38ab75/client/gutenberg/extensions/map/save.js#L25-L33) using `attributes` here?

Reply:
If we go the PHP route, can we use `esc_attr()` just to make it obvious that the value is OK in this context?

Reply:
> If we go the PHP route, can we use esc_attr() just to make it obvious that the value is OK in this context?
I said it first

Reply:
@ockham That will not work since the key can change in the future and we will not want to update every block in order to update that value.